src/main/groovy/swagger/grails/SwaggerGrailsPlugin.groovy

```diff
@@ -28,14 +28,18 @@ class SwaggerGrailsPlugin extends Plugin {
 
     Closure doWithSpring() {
         { ->
-            swagger(Swagger) { bean ->
-                basePath = grailsApplication.config.server.contextPath ?: null
-                securityDefinitions = ["apiKey": new ApiKeyAuthDefinition("apiKey", In.HEADER)]
-                security = [new SecurityRequirement().requirement("apiKey")]
-            }
-
-            swaggerCache(SwaggerCache) { bean ->
-                swagger = ref('swagger')
+            def conf = grailsApplication.config
+            def swaggerConfig = conf.swagger
+            if (swaggerConfig.active) {
```

Comment:
I think this should do the same check that is occurring in the controller to keep things consistent.

i.e. `Holders.config.getProperty....`